config/standalone-config.properties

```diff
@@ -10,38 +10,62 @@
 
 ##region General Segment Store Settings
 
-# Port on which a ZK server is started
+# Port on which a ZK server is started.
+# Default value: 4000
 #singlenode.zkPort=4000
 
 # Port on which segmentstore server is started.
+# Default value: 6000
 #singlenode.segmentstorePort=6000
 
 # Port on which the controller server grpc is started.
+# Default value: 9090
 #singlenode.controllerPort=9090
 
-# Enable REST server.
-#singlenode.enableRestServer=true
+# Whether to enable REST server.
+# Default value: true
+#singlenode.enableRestServer=false
```

Comment:
To be consistent with the configuration settings, we should probably use the default values for the new commented out settings.

e.g. change

```
# Default value: true
#singlenode.enableRestServer=false
```

to 

```
# Default value: true
#singlenode.enableRestServer=true
```